components/clp-package-utils/clp_package_utils/scripts/del_archives.py

```diff
@@ -57,6 +59,11 @@ def main(argv):
         logger.exception("Failed to load config.")
         return -1
 
+    storage_type = clp_config.archive_output.storage.type
+    if StorageType.FS != storage_type:
+        logger.error(f"Archive deletion is not supported for storage type: {storage_type}.")
```

Comment:
Not changing to "archive storage type" since we have "Archive" at the beginning of the sentence.